features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/SearchFragment.kt

```diff
@@ -534,11 +534,13 @@ class SearchFragment : Fragment(R.layout.fragment_search) {
         val searchLocation = if (viewModel.selectedTerritory.value?.isNotEmpty() == true) {
             viewModel.selectedTerritory.value
         } else {
-            val radiusOption = viewModel.selectedRadiusOption
-            resources.getQuantityString(
-                if (viewModel.isMetric) R.plurals.radius_kilometers else R.plurals.radius_miles,
-                radiusOption, radiusOption
-            )
+            val radiusOption = viewModel.selectedRadiusOption.value
+            radiusOption?.let {
```

Comment:
So the `searchLocation` is nullable now and we're calling `getString` with null potentially? This is wrong, need to use the default radius value if `radiusOption` is null.